ank-core/src/main/kotlin/io/kategory/ank/ank.kt

```diff
@@ -6,6 +6,7 @@ import java.io.File
 
 const val AnkBlock = ":ank"
 const val AnkSilentBlock = ":ank:silent"
+const val AnkCompileBlock = ":ank:compile"
```

Comment:
we can reuse `ank:silent` or remove it in favor of `ank:compile` since essentially both produce the same output which is guaranteeing compilation and being added to the scope of the page scripts but not actually evaluating anything.

Reply:
`silent` gets evaluated by `ScriptEngine` which means you can reference it in a regular `ank` blocks. Whatever is defined in `compile` cannot be referenced from a regular `ank` block.

Reply:
makes sense, let's keep them separate then.